src/index.umd.ts

```diff
@@ -10,13 +10,15 @@ import * as Controls from "./control";
 import * as Loaders from "./loader";
 import * as Plugins from "./plugin";
 import * as Constants from "./const/external";
+import * as Static from "./static";
 import { merge } from "./utils";
 
 merge(View3D, Core);
 merge(View3D, Controls);
 merge(View3D, Loaders);
 merge(View3D, Plugins);
 merge(View3D, Constants);
+merge(View3D, Static);
 (View3D as any).View3DError = View3DError;
```

Comment:
Is View3DError being exported in core or index.ts?

Reply:
Thanks for checking :+1:, that's an old code from v1 that should be removed.